smithy-openapi/src/test/resources/software/amazon/smithy/openapi/fromsmithy/protocols/examples-test.smithy

```diff
@@ -0,0 +1,213 @@
+           input: {
+                accountNumber: "102935",
+                username: "sichanyoo",
+                accountHistory: ["10", "-25", "50"],
+                depositAmount: "200"
+           },
+           output: {
+                username: "sichanyoo",
+                authenticationResult: ["pass1", "pass2", "pass3"],
+                textMessage: "You deposited 200-text",
+                emailMessage: "You deposited 200-email"
+           },
+      },
+
+      {
+           title: "Deposit invalid username example",
+           documentation: "depositTestDoc2",
+           input: {
+                username: "sichanyoo",
+                accountHistory: ["-200", "200", "10"],
+                depositAmount: "-200"
+           },
+           error: {
+                shapeId: InvalidUsername,
+                content: {
+                    internalErrorCode: "4gsw2-34",
+                    errorMessage: "ERROR: Invalid username."
+                }
+           },
+      },
+
+      {
+           title: "Deposit invalid amount example",
+           documentation: "depositTestDoc3",
+           input: {
+                accountNumber: "203952",
+                username: "obidos",
+                accountHistory: ["2000", "50000", "100"],
+                depositAmount: "-100"
+           },
+           error: {
+                shapeId: InvalidAmount,
+                content: {
+                    errorMessage1: "ERROR: Invalid amount.",
+                    errorMessage2: "2gdx4-34",
+                    errorMessage3: "2gcbe-98"
+                }
+           },
+      }
+  ]
+)
+
+apply Withdraw @examples(
+  [
+      {
+           title: "Withdraw valid example",
+           documentation: "withdrawTestDoc",
+           input: {
+                accountNumber: "124634",
+                username: "amazon",
+                withdrawParams: {"location" : "Denver", "bankName" : "Chase"},
+                time: "Tue, 29 Apr 2014 18:30:38 GMT",
+                withdrawAmount: "-35",
+                withdrawOption: "ATM"
+           },
+           output: {
+                branch: "Denver-203",
+                accountHistory: ["34", "5", "-250"],
+                location: "Denver",
+                bankName: "Chase",
+                atmRecording: "test"
+           },
+      },
+
+      {
+           title: "Withdraw invalid username example",
+           documentation: "withdrawTestDoc2",
+           input: {
+                accountNumber: "231565",
+                username: "peccy",
+                withdrawParams: {"location" : "Seoul", "bankName" : "Chase"},
+                withdrawAmount: "-450",
+                withdrawOption: "Venmo"
+           },
+           error: {
+                shapeId: InvalidUsername,
+                content: {
+                    internalErrorCode: "8dfws-21",
+                    errorMessage: "ERROR: Invalid username."
+                }
+           },
+      }
+  ]
```

Comment:
nit: looks like this file has some alignment issues

Reply:
alignment fixed.